README.md

```diff
@@ -12,3 +12,47 @@ This currently supports the following three formats commonly used in robotics:
 - [STL](https://en.wikipedia.org/wiki/STL_(file_format)) (.stl)
 - [COLLADA](https://en.wikipedia.org/wiki/COLLADA) (.dae)
 - [Wavefront OBJ](https://en.wikipedia.org/wiki/Wavefront_.obj_file) (.obj)
+
+# Usage
+[`Scene`] is filled with mesh data : name, vertices, normals, colors, texcoords and faces. But also its materials : opacity, shininess, index of refraction, texture, colors, ...
+
+[`Loader`] implements `load_{stl/collada/obj}` and `load_from_{stl/collada/stl}_slice`.
```

Comment:
Actually, `load` and `load_from_slice`, which determine the file type from the path or file content, are most easy to use APIs, so it might be a good idea to mention them and then mention the methods for the individual formats.